pyannotate_runtime/tests/test_collect_types.py

```diff
@@ -579,6 +579,22 @@ def test_skip_lambda(self):
             (lambda x, y: x+y)(0, 0)
         assert self.stats == []
 
+    def test_unknown_module_types(self):
+        # type: () -> None
+        def func_with_unknown_module_types(c):
+            # type: (Any) -> Any
+            return c
+
+        with self.collecting_types():
+            ns = {
+                '__name__': '<unknown>'
+            }
```

Comment:
To fix the mypy error from the tests, add `# type: Dict[str, Any]` here (and maybe add the necessary imports of Dict and Any.